test/e2e/parallel/proxy_test.go

```diff
@@ -985,7 +985,7 @@ func createAppStudioUser(t *testing.T, awaitilities wait.Awaitilities, user *pro
 		EnsureMUR().
 		RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin())...).
 		Execute(t)
-	user.signup, _ = req.Resources()
+	user.signup, _, _ = req.Resources(t)
 	user.token = req.GetToken()
 	tiers.MoveSpaceToTier(t, awaitilities.Host(), user.signup.Status.CompliantUsername, "appstudio")
```

Comment:
TBH, we could also drop the `Resources` method and return the values from the `Execute` one. IIRC, there was an idea that we could build some additional API on top of that (after calling `Execute`) but I doubt that we would need it something like that any time (we haven't needed it in the last few years).

Reply:
Thanks for noticing it! I will change it :)

Reply:
@MatousJobanek, I like your suggestion! I think it makes sense to drop `Resources`. If it was not needed in the last few years, maybe it will not be in the future. And if it is, we can always add it again